cli/command/service/create.go

```diff
@@ -74,6 +74,8 @@ func runCreate(dockerCli *command.DockerCli, opts *serviceOptions) error {
 
 	ctx := context.Background()
 
+	resolveServiceImageDigest(dockerCli, &service)
```

Comment:
Hmm. That's a tough one. I can see the security argument for doing it, since otherwise a denial of service can cause a downgrade to non-trusted resolution (by making it happen on the daemon). However, there are valid use cases that involve specifying images that have never been pushed to a registry, and the client won't be able to know if the images already exist on the worker nodes.

Should we require content trust to be turned off to support local image use cases?